Cargo.lock

```diff
@@ -12083,7 +12083,18 @@ version = "0.3.5"
 source = "git+https://github.com/aptos-labs/serde-reflection?rev=839aed62a20ddccf043c08961cfe74875741ccba#839aed62a20ddccf043c08961cfe74875741ccba"
 dependencies = [
  "once_cell",
- "serde 1.0.149",
+ "serde 1.0.183",
+ "thiserror",
+]
+
+[[package]]
+name = "serde-reflection"
+version = "0.3.6"
+source = "registry+https://github.com/rust-lang/crates.io-index"
```

Comment:
we're pulling in two versions of serde-reflection now, could we fix that -- would be curious if that might be the culprit here...

Reply:
The crates.io version also suffers from the same problem, and the test fails.

Reply:
Do you mean you assured both versions are the same? Not sure what you mean with 'the crates.io version', all those versions are from crates.io.